src/rgw/rgw_op.cc

```diff
@@ -2004,7 +2006,7 @@ int RGWGetObj::handle_slo_manifest(bufferlist& bl)
 
   vector<RGWAccessControlPolicy> allocated_acls;
   map<string, pair<RGWAccessControlPolicy *, boost::optional<Policy>>> policies;
-  map<string, rgw_bucket> buckets;
+  map<string, RGWBucketInfo*> bucket_infos;
```

Comment:
this map is storing pointers to `RGWBucketInfo bucket_info;` that goes out of scope. it looks like this needs to store them by value instead of pointer:
```c++
   map<string, RGWBucketInfo> bucket_infos;
```

Reply:
OK，i have fixed this bug.Thanks.

Reply:
@cbodley